brands/tourism/hotel.json

```
@@ -246,6 +246,16 @@
       "official_name": "Embassy Suites by Hilton",
       "tourism": "hotel"
     }
+  },
+    "tourism/hotel|EVEN Hotels": {
+    "countryCodes": ["us"],
+    "tags": {
+      "brand": "EVEN Hotels",
+      "brand:wikidata": "Q5416522",
+      "brand:wikipedia": "en:Even Hotels",
+      "name": "Even Hotels",
```

Match brand capitalization
```suggestion
      "name": "EVEN Hotels",
```